modules/images-editing-image-registry-settings-hcp.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * openshift_images/image-configuration-hcp.adoc
+// * post_installation_configuration/preparing-for-users.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="images-editing-image-registry-settings-hcp_{context}"]
+= Editing image registry settings for ROSA with HCP
+
+Description needed
```

Comment:
Update or edit the image registry for the cluster, you can run the following command: